dataland-frontend/src/components/forms/parts/fields/CurrencyDataPointFormField.vue

```diff
@@ -53,6 +53,9 @@ export default defineComponent({
   components: { NumberFormField, ExtendedDataPointFormField, UploadFormHeader, FormKit },
   props: {
     ...BaseFormFieldProps,
+    unit: {
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/bc6f54d6-1f8c-4ecb-b4ea-092e644556ff)
Input fields should be aligned

Reply:
fixed

Reply:
![image](https://github.com/d-fine/Dataland/assets/104447806/7f87069b-f984-45b4-bf42-a7db6fd1f80d)
Why are they now so different in width?

Reply:
For a moment I thought it would be nicer this way but ... No :) 
changed

Reply:
Is it possible to make the currency field slightly wider? Most of the currency currently are not completly displayed